src/config.rs

```diff
@@ -911,7 +911,7 @@ pub struct Config {
     pub local_dns_addr: Option<SocketAddr>,
     /// Remote DNS's address
     #[cfg(target_os = "android")]
-    pub remote_dns_addr: Option<String>,
+    pub remote_dns_addr: Option<Address>,
```

Comment:
Should be a `String` here, or `socks5::Address`.

The remote DNS address would be something like: `dns.google.com:53`

Reply:
Yes, it is an `socks5::Address`.